src/components/icon/_hatch.less

```diff
@@ -0,0 +1,51 @@
+.icon-hatch-btn {
+  --iconHatchCol: @black;
+  [data-theme="dark"] & {
+    .icon-hatch-dark();
+  }
+  @media (prefers-color-scheme: dark) {
+    .icon-hatch-dark();
+  }
+
+  height: 50px;
+  width: 50px;
+
+  border-radius: 100%;
+  border: solid 0px;
+
+  background: transparent;
+  color: var(--iconHatchCol);
```

Comment:
Instead of using CSS variables please use Less Variables and store all variables in _varibles.less. Make sure to name the variables with the prefix `hatch`. For examples `@hatchCol`